packages/config/config/devices/0x0039/templates/honeywell_template.json

```
@@ -0,0 +1,683 @@
+		"unit": "Cycles per hour"
+	},
+	"base_hvac_cycle_rate_12_default_9": {
+		"$import": "#base_hvac_cycle_rate_12",
+		"defaultValue": 9
+	},
+	"base_aux_em_heat_type": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Electric",
+				"value": 0
+			},
+			{
+				"label": "Gas/Oil",
+				"value": 1
+			}
+		]
+	},
+	"base_lockout_temperature": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 65,
+		"defaultValue": 65,
+		"unit": "°F",
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			}
+		]
+	},
+	"base_air_filter_reminder": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 19,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "10 Run time days",
+				"value": 1
+			},
+			{
+				"label": "20 Run time days",
+				"value": 2
+			},
+			{
+				"label": "30 Run time days",
+				"value": 3
+			},
+			{
+				"label": "45 Run time days",
+				"value": 4
+			},
+			{
+				"label": "60 Run time days",
+				"value": 5
+			},
+			{
+				"label": "90 Run time days",
+				"value": 6
+			},
+			{
+				"label": "120 Run time days",
+				"value": 7
+			},
+			{
+				"label": "150 Run time days",
+				"value": 8
+			},
+			{
+				"label": "30 Days",
+				"value": 9
+			},
+			{
+				"label": "45 Days",
+				"value": 10
+			},
+			{
+				"label": "60 Days",
+				"value": 11
+			},
+			{
+				"label": "75 Days",
+				"value": 12
+			},
+			{
+				"label": "3 Months",
+				"value": 13
+			},
+			{
+				"label": "4 Months",
+				"value": 14
+			},
+			{
+				"label": "5 Months",
+				"value": 15
+			},
+			{
+				"label": "6 Months",
+				"value": 16
+			},
+			{
+				"label": "9 Months",
+				"value": 17
+			},
+			{
+				"label": "12 Months",
+				"value": 18
+			},
+			{
+				"label": "15 Months",
+				"value": 19
+			}
+		]
+	},
+	"base_reminder_months": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 12,
+		"defaultValue": 0,
+		"unit": "months",
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			}
+		]
+	},
+	"base_reminder_months_by_3": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 12,
+		"defaultValue": 0,
+		"unit": "months",
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "3",
+				"value": 3
+			},
+			{
+				"label": "6",
+				"value": 6
+			},
+			{
+				"label": "9",
+				"value": 9
+			},
+			{
+				"label": "12",
+				"value": 12
+			}
+		]
+	},
+	"base_reminder_months_6_12_24": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 24,
+		"defaultValue": 0,
+		"unit": "months",
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "6",
+				"value": 6
+			},
+			{
+				"label": "12",
+				"value": 12
+			},
+			{
+				"label": "24",
+				"value": 24
+			}
+		]
```

```suggestion
		"allowManualEntry": false,
		"options": [
			{
				"label": "Disable",
				"value": 0
			},
			{
				"label": "6 months",
				"value": 6
			},
			{
				"label": "12 months",
				"value": 12
			},
			{
				"label": "24 months",
				"value": 24
			}
		]
```

I don't think UIs add the unit to dropdowns, so better put them here.